packages/grid/_modules_/grid/components/containers/GridRoot.tsx

```diff
@@ -4,47 +4,32 @@ import { visibleColumnsLengthSelector } from '../../hooks/features/columns/colum
 import { useGridSelector } from '../../hooks/features/core/useGridSelector';
 import { useGridState } from '../../hooks/features/core/useGridState';
 import { classnames } from '../../utils';
-import { getCurryTotalHeight } from '../../utils/getTotalHeight';
 import { ApiContext } from '../api-context';
 
-export interface GridRootProps extends React.HTMLAttributes<HTMLDivElement> {
-  size: { width: number; height: number };
-  header: React.RefObject<HTMLDivElement>;
-  footer: React.RefObject<HTMLDivElement>;
```

Comment:
These two props were bleeding into the DOM, see in production: https://material-ui.com/components/data-grid/#mit-version

<img width="194" alt="Capture d’écran 2021-01-27 à 23 31 22" src="https://user-images.githubusercontent.com/3165635/106062965-ca2bf880-60f7-11eb-9f4a-a9729810c2b2.png">

https://material-ui.com/components/data-grid/#mit-version